packages/bridge/src/skip/__tests__/skip-bridge-provider.spec.ts

```diff
@@ -431,4 +540,109 @@ describe("SkipBridgeProvider", () => {
 
     expect(approvalTxRequest).toBeUndefined();
   });
+
+  describe("getAvailableSourceAssetVariants", () => {
+    it("gets shared origin assets", async () => {
+      const sourceVariants = await provider.getAvailableSourceAssetVariants(
+        {
+          chainId: "osmosis-1",
+          chainType: "cosmos",
+        },
+        {
+          denom: "USDC",
+          address:
+            "ibc/498A0751C798A0D9A389AA3691123DADA57DAA4FE165D5C75894505B876BA6E4",
+          decimals: 6,
+          sourceDenom: "uusdc",
+        }
+      );
+
+      expect(sourceVariants).toEqual([
+        {
+          address: "uusdc",
+          chainId: "noble-1",
+          chainType: "cosmos",
+          decimals: 6,
+          denom: "USDC",
+          sourceDenom: "uusdc",
+        },
+        {
+          address: "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+          chainId: 1,
+          chainType: "evm",
+          decimals: 6,
+          denom: "USDC",
+          sourceDenom: "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+        },
+        {
+          address:
+            "ibc/FE98AAD68F02F03565E9FA39A5E627946699B2B07115889ED812D8BA639576A9",
+          chainId: "agoric-3",
+          chainType: "cosmos",
+          denom: "USDC",
+          sourceDenom: "uusdc",
+          decimals: 6,
+        },
+        {
+          address:
+            "ibc/43897B9739BD63E3A08A88191999C632E052724AB96BD4C74AE31375C991F48D",
+          chainId: "archway-1",
+          chainType: "cosmos",
+          denom: "USDC",
+          sourceDenom: "uusdc",
+          decimals: 6,
+        },
+      ]);
+    });
+
+    it("includes skip supported cosmos counterparty assets from asset list", async () => {
+      const sourceVariants = await provider.getAvailableSourceAssetVariants(
+        {
+          chainId: "osmosis-1",
+          chainType: "cosmos",
+        },
+        {
+          denom: "USDC",
+          address:
+            "ibc/498A0751C798A0D9A389AA3691123DADA57DAA4FE165D5C75894505B876BA6E4",
+          decimals: 6,
+          sourceDenom: "uusdc",
+        }
+      );
+
+      expect(sourceVariants[0]).toEqual({
+        address: "uusdc",
+        chainId: "noble-1",
+        chainType: "cosmos",
+        decimals: 6,
+        denom: "USDC",
+        sourceDenom: "uusdc",
+      });
+    });
```

Comment:
Wasn't this tested in the previous test? If we want to emphasize it, how about adding a comment or an assertion there?